spec/index.html

```diff
@@ -734,9 +735,153 @@ <h3>Overview</h3>
     <section id="hash-1d-quads-exampkes" class="informative">
       <h3>Examples</h3>
 
-      <p class="ednote">
-        Add some examples where the algorithm results are unique or not.
-      </p>
+      <aside id="ex-unique-hashes"
+            class="example"
+            title="Unique hashes">
+        <p>This example illustrates hashing quads containing
+          blank nodes where hashing the statements mentioning
+          those blank nodes generates unique results.</p>
+
+        <figure id="fig-unique-hashes" style="text-align:center">
+          <!-- Source for this file is at https://docs.google.com/drawings/d/1LQ6fp4a35lrEOKRMD20gcncUkoitX0eW8QI_U2mQnAs/edit?usp=sharing -->
+          <object data="unique-hashes.svg" style="width: 75%" type="image/svg+xml" aria-describedby="fig-unique-hashes-alt">
+            <p id="fig-unique-hashes-alt">
+              The image represents the graph described in
+              <a href="#ex-unique-hashes-input">
+                the following code block
+              </a>.</p>
+          </object>
+          <figcaption>An illustration of a graph resulting in unique hashes.<br/>
+            Image available in
+            <a href="unique-hashes.svg" title="SVG image of a unique hashes">
+              <abbr title="Scalable Vector Graphics">SVG</abbr>
+            </a>.</figcaption>
+        </figure>
+
+        <pre id="ex-unique-hashes-input"
+             data-transform="updateExample"
+             title="Graph with blank nodes resulting in unique hashes">
+          <!--
+            :p :q _:b0 .
+            :p :r _:b1 .
+            _:b0 :p _:b2 .
+            _:b1 :p _:b2 .
+          -->
+        </pre>
+
+        <p>The algorithm will be called twice, with each blank node (`_:b0` and `_:b1`).
```

Comment:
There's a third blank node here `_:b2`. Should that instead be a non-blank node in this example?

Reply:
You're correct; the example evolved from the original I had. But `_:b0 :p _:b2` and `_:b1 :p _:b2` would exhibit the more complicated scenario of requiring an N-Degree hash, which I was introducing in the next example. Better to replace the graph with something not requiring an N-Degree hash.